blazor/getting-started/maui-blazor-app.md

```diff
@@ -158,15 +129,15 @@ Before running the sample, make sure the mode is `Windows Machine`.
 
 ![Build and run MAUI Blazor App](images/maui/windows-machine-mode.png)
 
-N> If you want to run the application in Android or iOS refer [MAUI Getting Started](https://learn.microsoft.com/en-us/dotnet/maui/get-started/first-app?tabs=vswin&pivots=devices-android) for the setup. 
+N> If you want to run the application in Android or iOS refer [MAUI Getting Started](https://learn.microsoft.com/en-us/dotnet/maui/get-started/first-app?tabs=vswin&pivots=devices-android) for the setup.
```

Comment:
The link is incorrect

Reply:
Correct links was updated in mentioned place.